src/cheri_pte.sail

```diff
@@ -42,98 +42,62 @@ union PTE_Check = {
   PTE_Check_Failure : ext_ptw
 }
 
-function to_pte_check(b : bool) -> (bool, pte_check) =
-  (b, PTE_CAP_OK)
-
 function checkPTEPermission(ac : AccessType(ext_access_type), priv : Privilege, mxr : bool, do_sum : bool, p : PTE_Bits, ext : extPte, ext_ptw : ext_ptw) -> PTE_Check = {
-  let (succ, pte_chk) : (bool, pte_check) =
+  /*
+   * Although in many cases MXR doesn't make sense for capabilities, we honour
+   * it for three reasons:
+   *
+   * 1. It provides uniformity rather than giving strange and surprising edge cases.
+   *
+   * 2. The tag-dependence of loads is achieved by passing Data in for untagged
+   *    capabilities. Thus, not honouring MXR for capabilities would result in
+   *    differences in whether MXR had an effect based on the tag.
+   *
+   * 3. It's simpler to implement yet still safe (LC is unaffected by MXR).
```

Comment:
I should probably change this to be clear that I mean the LC _bit_, because LC-the-instruction is definitely affected by MXR, that's what the whole discussion is about.